
    ********************************************************************************
    Date/Time: 2023-11-24T09:06:07.173Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"ENOENT: no such file or directory, open './db/product-bad.json'","stack":"Error: ENOENT: no such file or directory, open './db/product-bad.json'","originalError":{"errno":-2,"code":"ENOENT","syscall":"open","path":"./db/product-bad.json"},"requestInfo":{"hostname":"localhost","path":"/api/product"}}
    ******************************************************************************** 
    ********************************************************************************
    Date/Time: 2023-11-29T05:45:10.588Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"repo.getById is not a function","stack":"TypeError: repo.getById is not a function\n    at /Users/hjalte/Projects/NodeApi/routes/product.js:67:7\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:284:15\n    at param (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:365:14)\n    at param (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:376:14)\n    at Function.process_params (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:421:3)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:280:10)","originalError":{},"requestInfo":{"hostname":"localhost","path":"/api/product/706"}}
    ******************************************************************************** 
    ********************************************************************************
    Date/Time: 2023-11-29T07:28:28.227Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"repo.getById is not a function","stack":"TypeError: repo.getById is not a function\n    at /Users/hjalte/Projects/NodeApi/routes/product.js:67:7\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:284:15\n    at param (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:365:14)\n    at param (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:376:14)\n    at Function.process_params (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:421:3)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:280:10)","originalError":{},"requestInfo":{"hostname":"localhost","path":"/api/product/706"}}
    ******************************************************************************** 
    ********************************************************************************
    Date/Time: 2023-11-29T07:28:30.883Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"sqlite3 is not defined","stack":"ReferenceError: sqlite3 is not defined\n    at db.submit (/Users/hjalte/Projects/NodeApi/helpers/sqlserver.js:11:18)\n    at Object.get (/Users/hjalte/Projects/NodeApi/repositories/product-sqlServer.js:12:12)\n    at /Users/hjalte/Projects/NodeApi/routes/product.js:6:25\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/hjalte/Projects/NodeApi/node_modules/express/lib/router/index.js:280:10)","originalError":{},"requestInfo":{"hostname":"localhost","path":"/api/product"}}
    ******************************************************************************** 
    ********************************************************************************
    Date/Time: 2023-11-29T07:30:23.854Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"SQLITE_RANGE: column index out of range","stack":"Error: SQLITE_RANGE: column index out of range","originalError":{"errno":25,"code":"SQLITE_RANGE"},"requestInfo":{"hostname":"localhost","path":"/api/product"}}
    ******************************************************************************** 
    ********************************************************************************
    Date/Time: 2023-11-29T07:30:51.160Z
    Exception Info: {"status":500,"statusText":"Internal Server Error","message":"SQLITE_RANGE: column index out of range","stack":"Error: SQLITE_RANGE: column index out of range","originalError":{"errno":25,"code":"SQLITE_RANGE"},"requestInfo":{"hostname":"localhost","path":"/api/product"}}
    ******************************************************************************** 